lib/players/computer.rb

```diff
@@ -0,0 +1,32 @@
+module Players
+  class Computer < Player
+    def move(board)
+      move = nil
+      # first move, take the middle square
+      if !board.taken?(5)
+        move = "5"
+      # second move, take the top left corner
+      elsif board.turn_count == 1
+        move = "1"
+      # third move, check top right and bottom left corner
+      elsif board.turn_count == 2
+        move = [3, 7].detect{|i| !board.taken?(i)}.to_s
+      # Check for any combinations of 2 matching tokens and an empty square  
+      else
+        Game::WIN_COMBINATIONS.detect do |combo|
```

Comment:
You seem to be calling `board.position(i+1)` quite often, it might be worth encapsulating that to a helper method e.g.
``` ruby
def position(i)
 board.position(i+1)
end
```
Same with `board.taken?(i+1)`

Also, instead of setting `move` to `nil` and then conditionally resetting it, it might be worth just using the explicit return within the appropriate conditionals, so that if you ever reach the point where you're resetting `move` just `return` it. (even though we try to avoid using explicit return if possible this would dry up the method considerably)